app/util/job.ts

```diff
@@ -161,23 +161,20 @@ export async function pauseAndSaveJob(
 }
 
 /**
- * Resume a paused job then save it.
+ * Updates the user access token in the database and applies the appropriate
+ * job function - either resume or skipPreview.
```

Comment:
Is there a valid use case for calling this function with neither 'resume' nor 'skipPreview'? If not, I think doing dispatch using a string argument is not great. Better to have two separate functions for each functionality that call a common 'updateAccessToken' function to do the common work.

Reply:
Refactored.